src/modules/mavlink/mavlink_params.c

```diff
@@ -156,15 +156,6 @@ PARAM_DEFINE_INT32(MAV_HB_FORW_EN, 1);
  */
 PARAM_DEFINE_INT32(MAV_RADIO_TOUT, 5);
 
-/**
```

Comment:
Are these HITL changes related to this topic or ?

Reply:
No, it was a revert to be able to run this in sitl for now